COPY/etc/manageiq/postgresql.conf.d/01_miq_overrides.conf

```diff
@@ -36,7 +39,9 @@ max_worker_processes = 10
 wal_level = 'logical'
 wal_log_hints = on
 wal_buffers = 16MB
-checkpoint_completion_target = 0.9
+min_wal_size = 1GB
+max_wal_size = 2GB
+checkpoint_completion_target = 0.7
```

Comment:
> checkpoint_completion_target = 0.7 : this can be increased back up to 0.9, however changes to checkpoint_segments will be recommended

I think I'd rather keep this at 0.9 and `checkpoint_segments` was removed in 9.5

Reply:
👍